### PR TITLE
chore: webui update TableBatch component to show # of selected items [DET-4047]

### PR DESCRIPTION
**Pull request by apizzini**
## Description

Update table batch component to show # of selected items.

## Test Plan

Checked working app and working storybook.

## Commentary (optional)

Screenshots:
Schermata 2020-09-04 alle 14 18 10
Schermata 2020-09-04 alle 14 18 04

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-4047]: https://determinedai.atlassian.net/browse/DET-4047

### CODE REVIEW

File: webui/react/src/components/TableBatch.stories.tsx [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+  </TableBatch>
+);
+
+export const Custom = (): React.ReactNode => (
+  <TableBatch selectedRowCount={number('selectedRowCount', 1)}>
+    <Button danger type="primary">Batch Operation 1</Button>
+    <Button>Batch Operation 2</Button>
+    <Button>Batch Operation 3</Button>
+  </TableBatch>
+);
```
Comment (hkang1) [inline webui/react/src/components/TableBatch.stories.tsx:27]: nice!

File: webui/react/src/components/TableBatch.tsx [hunk 2]
```diff
@@ -4,25 +4,28 @@ import css from './TableBatch.module.scss';
...
+  selectedRowCount: 0,
 };
 
 const TableBatch: React.FC<Props> = (props: PropsWithChildren<Props>) => {
   const classes = [ css.base ];
+  const selectedRowCount = props.selectedRowCount || 0;
 
-  if (props.show) classes.push(css.show);
+  const message = 'Apply batch operations to '+selectedRowCount+
+    ' item'+(selectedRowCount===1?'':'s');
```
Comment (hkang1) [inline webui/react/src/components/TableBatch.tsx:20]: **nit:** template literals
```
const message = `Apply batch operations to ${selectRowCount} item${selectedRowCount === 1 ? '' : 's'}`;
```

**Review by hkang1 — APPROVED** [review]
LGTM!

File: webui/react/src/components/TableBatch.tsx (hunk 2 above)
Comment (apizzini) [reply webui/react/src/components/TableBatch.tsx]: Updated.

**Pull request merged**